### DISCUSSION TITLE
Problem with uncle blocks

### DISCUSSION THREAD

**Discussion by SuperMinerHero** [coinfoundry/miningcore#1164] [Q&A]
Hello
I am just testing the miningcore on the ropsten network and I noticed a problem with uncle's blocks.
The first time the uncle's block became an orphan, and the next time the block was duplicated, a record in the database.
![image](https://user-images.githubusercontent.com/99591742/153750624-d2c1e283-791f-4bde-a071-12a76a1dbef3.png)

![image](https://user-images.githubusercontent.com/99591742/153750648-4f7140a6-366b-487f-ae47-f541906a924d.png)
Is this some configuration error?

**Comment by SuperMinerHero** [comment]
Sometimes the rewards are correct:
![image](https://user-images.githubusercontent.com/99591742/153753569-d6a5f9db-5769-40e8-a593-195b5400888d.png)

After removing duplicates for uncle, the reward is like a full block:
![image](https://user-images.githubusercontent.com/99591742/153753845-5e65f68c-38e4-449d-9019-f2642954b6d7.png)
At etherscan it is uncle block:
https://ropsten.etherscan.io/uncle/0xdd79b13ff370a9a2f460aee5504b9a5e927436ea8f8eaf070d6987ff0d021e03

**Comment by ei8ht187** [comment]
Seems to be the same issue as discussed in #1103 and in issue #1080.
@oliverw maybe reopen the issue #1080?
